### ISSUE TITLE
[Bug]: CLI init for Magento integration (manual install) fails (Windows+WSL)

### ISSUE THREAD

**Issue by DavidDeSloovere** [vuestorefront/vue-storefront#6919]
### Describe the Bug

Once again it's not possible to create a project using the CLI. 

Throws error when trying to delete .env.example

### Current behavior

```
    Error: ENOENT: no such file or directory, rmdir 'D:\Temp\vsfmag\whyfail\.env.example'
    Code: ENOENT
```

The file exists, but it's not a directory. `rmdirSync` is used, which might be the reason it fails?
Shoud it be `unlinkSync`

https://github.com/vuestorefront/vue-storefront/blob/4e1ff1aea26493574db860b845128a0ec042b448/packages/cli/src/domains/magento2/functions/copyEnv.ts#L34

### Expected behavior

No errors.

### Steps to reproduce

On Windows or WSL:

npx @vue-storefront/cli@latest init
using Magento 2 and No auto installation

![image](https://github.com/vuestorefront/vue-storefront/assets/352626/d58578b4-7a80-4a48-8753-772f3f71d40e)

### What version of Vue Storefront are you using?

-

### What version of Node.js are you using?

16.20

### What browser (and version) are you using?

-

### What operating system (and version) are you using?

Windows / WSL

### Relevant log output

```shell
Error: ENOENT: no such file or directory, rmdir 'D:\Temp\vsfmag\whyfail\.env.example'
    Code: ENOENT
```

### Able to fix / change the documentation?

- [X] Yes
- [ ] No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by skirianov** [comment]
looking into it

**Comment by skirianov** [comment]
can you try to run npx @vue-storefront/cli@4.2.2-rc.1 init ? 

I just published release candidate package

**Comment by skirianov** [comment]
this should fix it #6924

**Comment by DavidDeSloovere** [comment]
👍
Fixed for WSL 
On Win another bug surfaces, but this one looks fixed.
